2021/statement_of_interest.md

```diff
@@ -0,0 +1,47 @@
+# Creating a statement of interest
+
+Based in the [official GSoD resources](https://developers.google.com/season-of-docs/docs/tech-writer-statement)
+
+In the 2021 Season of Docs program, organizations work directly with technical writers to create, improve, and update documentation.
+Organizations select technical writers directly.
+
+Technical writers interested in working with us should submit a statement of interest by mail to `arviz.devs@gmail.com`.
+Statements of interest shared on Gitter, as PRs submitted here or in any other channel will not be considered.
+
+Your statement of interest should include your contact information,
+information about your technical writing experience,
+and an outline of the work you would do with the organization.
+
+It should also highlight how you meet any specific requirements and skills in our [selection criteria]().
+Read the [project scope page]() carefully to make sure you have included all the required information.
+
+This template is an outline to help you create your own statement of interest.
+
+## Personal information
+
+Include your name and email address.
+If you do not wish your name included in the public final case study reports,
+please note that in your proposal and provide a username.
+
+## Professional information
+
+* A link to a publicly accessible portfolio of writing samples, or individual links to examples of your work.
+* An overview of your recent technical writing experience:
+   * List your most recent technical writing roles or work experience. Summarize your role, the responsibilities of the role, and the work you completed. Provide dates where relevant.
+   * If you have open source documentation experience (in any language, not necessarily Python/Julia), include it here too.
+* A link to your resume or curriculum vitae (CV).
+* (Optional) Any other information you want to add. For example, your interest in technical documentation, why you want to participate in Season of Docs, or other relevant experience.
+
+## Project statement
+
+* Project title.
+* Detailed description.
+   * Give a description of how you will approach the project
+   * Say how what you’re proposing will solve our problem or meet our goals
+   * Include a summary of any discussions you’ve had with us about the project.
```

Comment:
Do we need that? We'll already know? But maybe it's good to have it written down too?

Reply:
Agreed

Reply:
The summary? I do not think we need that, the other two points are fine.

Reply:
yes, that was only the summary, github is very confusing on what comments are about a single line and which are about multiple lines